src/ShapeCrawler/Drawing/SCColor.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Drawing;
```

Comment:
Unfortunately, we can't use `System.Drawing` namespace because it [only supports Windows starting from .NET 6](https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only). That's why we recently switched to [SkiaSharp](https://github.com/mono/SkiaSharp).

Reply:
I removed System.Drawing references.